### PR TITLE
Fix a number of Codacy warnings

### PR DESCRIPTION
**Pull request by vasily-v-ryabov**

### CODE REVIEW

**Comment by vasily-v-ryabov** [comment]
This pull request **fixes 3 alerts** when merging 1ce0de080b94d57c5defc30ef601eed54d2f4721 into 7f06f242b5721ff9b87f4a1473afd232c8f2f558 - [view on LGTM.com](https://lgtm.com/projects/g/pywinauto/pywinauto/rev/pr-7dc4168f31fca498ed4b49a70a2fdc6f9a28f780)

**fixed alerts:**

* 3 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: pywinauto/controls/hwndwrapper.py [hunk 1]
```diff
@@ -737,8 +735,8 @@ def wait_for_idle(self):
         win32functions.WaitGuiThreadIdle(self.handle)
 
     # -----------------------------------------------------------
-    def click(
-        self, button = "left", pressed = "", coords = (0, 0), double = False, absolute = False):
+    def click(self, button = "left", pressed = "", coords = (0, 0),
+              double = False, absolute = False):
         """
```
Comment (airelil) [inline pywinauto/controls/hwndwrapper.py]: PEP-8 recommends no spaces in parameters default values. https://www.python.org/dev/peps/pep-0008/#other-recommendations

File: pywinauto/controls/win32_controls.py [hunk 2]
```diff
@@ -212,12 +209,13 @@ def is_dialog(self):
         return False
 
     #-----------------------------------------------------------
-    def click(self, *args, **kwargs):
+    def click(self, button = "left", pressed = "", coords = (0, 0),
+              double = False, absolute = False):
```
Comment (airelil) [inline pywinauto/controls/win32_controls.py]: Same as above: no spaces around assignment operator for a default value

**Comment by vasily-v-ryabov** [comment]
This pull request **fixes 3 alerts** when merging e96cb7e524016eea7bf2b5553252d488a52df531 into 020193e3ee1a4a63ce89aaf236c9b37662eae92a - [view on LGTM.com](https://lgtm.com/projects/g/pywinauto/pywinauto/rev/pr-16f11bc761f8556c939a24bea8b948e0e47b4ef5)

**fixed alerts:**

* 3 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**